node/node.go

```diff
@@ -330,7 +326,7 @@ func (n *Node) startRPC() ([]net.Listener, error) {
 	listeners := make([]net.Listener, len(listenAddrs))
 	for i, listenAddr := range listenAddrs {
 		mux := http.NewServeMux()
-		wm := rpcserver.NewWebsocketManager(rpccore.Routes, n.evsw)
+		wm := rpcserver.NewWebsocketManager(rpccore.Routes)
```

Comment:
like here `rpccore.SetEventBus(n.eventBus)` ? and just let the websockets access it globally like everything else is global in that package? 

I guess that could be fine.